lua/zk/pickers/minipick.lua

```diff
@@ -0,0 +1,70 @@
+local M = {}
+local H = {}
+local minipick = require("mini.pick")
+
+M.note_picker_list_api_selection = { "title", "path", "absPath" }
+
+M.show_note_picker = function(notes, opts, cb)
+  notes = vim.tbl_map(function(n)
+    return { text = n.title, path = n.absPath, note = n }
+  end, notes)
+
+  opts = opts or {}
+  local minipick_opts = vim.tbl_deep_extend("force", {
+    source = {
+      items = notes,
+      name = opts.title,
+      choose = function(_)
+        return nil
+      end,
```

Comment:
what does this do?

Reply:
The `choose` function in `mini.pick` is used to take action on an item—such as opening a buffer to edit a file. Because `zk-nvim` already handles what to do with selected options, I set this to `nil`. The value selected is returned as a result of `MiniPick.start()`. I then pass that value to the provided `zk-nvim` callback to handle the "action" of what to do.

Reply:
Why do you need to specify it then? The default is already `choose = nil`. Does that not work for some reason?

Reply:
There is a difference between a `nil` value for `choose` and a function that returns `nil`. The former invokes the default action from `mini.pick` while the latter tells `mini.pick` to close the picker without executing an action. But as I mentioned in further comment, I'll be refactoring this.